flake.nix

```diff
@@ -0,0 +1,28 @@
+{
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/24.05";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs =
+    { self
+    , nixpkgs
+    , flake-utils
+    ,
+    }:
+    flake-utils.lib.eachDefaultSystem (
+      system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+      in
+      {
+        devShells.default = pkgs.mkShell {
+          nativeBuildInputs = with pkgs.buildPackages; [
+            python3
+            dmg2img
```

Comment:
I recommend adding 'libguestfs' here - it is required to rebuild the OpenCore image as described in https://github.com/kholia/OSX-KVM/blob/master/OpenCore/README.md#notes